hummingbot/strategy/arbitrage/arbitrage.pyx

```diff
@@ -501,7 +501,7 @@ cdef class ArbitrageStrategy(StrategyBase):
             total_ask_value_adjusted += ask_price_adjusted * amount
             # accumulated profitability with fees
             profitability = (total_bid_value_adjusted * (1 - sell_fee.percent) - total_sell_flat_fees) / \
-                            (total_ask_value_adjusted * (1 + buy_fee.percent) + total_buy_flat_fees)
+                            (total_ask_value_adjusted / (1 - buy_fee.percent) + total_buy_flat_fees)
```

Comment:
I can't really grasp how this works... please add some documentations, and maybe split up the calculation to multiple parts.

Reply:
https://docs.google.com/spreadsheets/d/1AU2GDfG7L4DO3xgbpLqiUlWjP2hmtO7pa4YScz0Q7qM/edit#gid=1145991255

Reply:
ill try to summarize this in the PR too